indigo/management/commands/datamigrationimportdocuments.py

```diff
@@ -0,0 +1,130 @@
+import json
+
+import jsondataferret
+from django.core.management.base import BaseCommand
+from jsondataferret.pythonapi.newevent import NewEventData, newEvent
+
+from indigo.models import Project
+
+
+class Command(BaseCommand):
+    help = "Data Migration - Import Documents"
+
+    def add_arguments(self, parser):
+        parser.add_argument("filename")
+
+    def handle(self, *args, **options):
+        with open(options["filename"]) as fp:
+            all_datas = json.load(fp)
+
+        writes = []
+
+        for data in all_datas:
+            documents = []
+            for related_document in data.get("related_documents"):
+                if _include_document(related_document.get("title")):
+                    documents.append(
+                        {
+                            "title": {"value": related_document.get("title")},
+                            "url": {
+                                "value": "https://golab.bsg.ox.ac.uk"
```

Comment:
Does this need a trailing slash, or does related_document.get("url") always start with one?

Reply:
the latter